### PR TITLE
Separate screening api calls from screening endpoints

### PR DESCRIPTION
**Pull request by ThomasOSI**
[#139486493]

### CODE REVIEW

File: app/controllers/api/v1/screenings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      end
+
+      def edit
+        @screening = ScreeningRepository.find(params[:id])
+        @participants = @screening.participants.to_a
+      end
+
+      def show
+        @screening = ScreeningRepository.find(params[:id])
+
```
Comment (NateOSI) [inline app/controllers/api/v1/screenings_controller.rb]: newline

File: app/controllers/api/v1/screenings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      end
+
+      def update
+        existing_screening = Screening.new(screening_params.to_h)
+        updated_screening = ScreeningRepository.update(existing_screening)
+        render json: updated_screening
+      end
+
+      def edit
+        @screening = ScreeningRepository.find(params[:id])
```
Comment (NateOSI) [inline app/controllers/api/v1/screenings_controller.rb]: the edit action doesn't render json?

File: config/routes.rb [hunk 3]
```diff
@@ -8,12 +8,17 @@
     constraints: Routes::InactiveReleaseOneConstraint
 
   resources :screenings,
-    only: [:create, :edit, :show, :update, :index],
+    only: [:show, :edit, :index],
```
Comment (NateOSI) [inline config/routes.rb]: missing index action

File: app/controllers/api/v1/screenings_controller.rb (hunk 1 above)
Comment (ThomasOSI) [inline app/controllers/api/v1/screenings_controller.rb]: removed

File: config/routes.rb (hunk 3 above)
Comment (ThomasOSI) [inline config/routes.rb]: Added to controller

File: app/controllers/api/v1/screenings_controller.rb (hunk 2 above)
Comment (ThomasOSI) [inline app/controllers/api/v1/screenings_controller.rb]: Correct.  Removed from controller as it no longer does anything meaningful.

**Pull request merged**